docs/app-guide/ingress.rst

```diff
@@ -0,0 +1,69 @@
+###################
+Defining an Ingress
```

Comment:
This is overly pedantic, but we're trying to write titles in sentence case based on [Google's style guide](https://developers.google.com/style/highlights).

Reply:
Oh, interesting! I'd never heard of that before, but https://atmosol.com/resource-post/headline-capitalization-ecommerce-title-case-vs-sentence-case/ has a pretty good reason for that. I'll also fix the Vault operational documentation and some of the other stuff I've written.

Do we do this for tech notes as well? I don't remember seeing it in the style guide, but maybe I missed it.